core/templates/pages/subtopic-viewer-page/subtopic-viewer-page.component.css

```diff
@@ -91,17 +91,22 @@
   }
 
   .oppia-subtopic-title {
-    font-size: 2em;
+    font-size: 32px;
```

Comment:
It is better to have relative sizing like 'rem' and 'em' than px. Is this change necessary?

Reply:
2em appears too large on smaller screen sizes, therefore I used px (below 600px)

Reply:
@Rijuta-s PTAL

Reply:
What about using rem? I believe it will not break on mobile screen size!

Reply:
Fixed (using 2 rem)

Reply:
Thanks!